public/components/player/list/list.js

```diff
@@ -9,17 +39,53 @@ require("can/view/href/");
 
 var Player = require("bitballs/models/player");
 
-var ViewModel = exports.ViewModel = CanMap.extend({
+var ViewModel = exports.ViewModel = CanMap.extend(
+/** @prototype */
+{
 	define: {
+		/**
+		 * @property {Boolean} bitballs/components/player/list.isAdmin isAdmin
+		 * @parent bitballs/components/player/list.properties
+		 *
+		 * Configures whether or not admin specific features are enabled.
+		 **/
+		isAdmin: {
+			type: 'boolean',
+			value: false
+		},
+		/**
+		 * @property {bitballs/models/player} bitballs/components/player/list.players players
+		 * @parent bitballs/components/player/list.properties
+		 *
+		 * A [bitballs/models/player] model List.
+		 */
 		players: {
 			value: function(){
 				return Player.getList({orderBy: "name"});
 			}
 		}
 	},
+	/**
+	 * @function editPlayer
+	 *
+	 * Selects a [bitballs/models/player] model for editing.
+	 *
+	 * @param {bitballs/models/player} player
+	 *   The player model that will be passed to the `<player-edit>`
+	 *   component.
+	 *
+	 * @return {undefined}
```

Comment:
@return {undefined} isn't needed.  In this is assumed if there is not a return specified.